sphinxcontrib/opencontracting.py

```diff
@@ -211,11 +213,104 @@ def run(self):
         return [admonition_node]
 
 
+class WorkedExampleListNode(nodes.General, nodes.Element):
```

Comment:
Docutils convention is to name node classes in lowercase (`paragraph`, etc.) with underscores (`definition_list_item`).

```suggestion
class worked_example_list(nodes.General, nodes.Element):
```